src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCustomAnnotation.java

```diff
@@ -83,8 +83,8 @@
         @Override //indent:8 exp:8
         public String toString() { //indent:8 exp:8
             return //indent:12 exp:12
-                    new String( //indent:20 exp:20
-                            new StringBuilder("Hello") //indent:28 exp:28
+                    new String( //indent:20 exp:12,16 warn
```

Comment:
linewrapping is 4.
So should be `//indent:20 exp: 16 warn`

we should not suggest indentation of `12` as it will be violation.

Reply:
Yes, but earlier it was explictly coded like that to follow the current indent of parent, and then lineWrpping also (multi-indents) so left that logic.
In some cases it is good for code readability too, to indent new keyword just down its parent.
Though I could change the logic to strictly follow only lineWrapping.

Reply:
But yes, there is some problem...Corrected Now.
Done.

Reply:
now it is much better.
```
        checkConfig.addAttribute("lineWrappingIndentation", "4");
        checkConfig.addAttribute("forceStrictCondition", "true");
```
so warn after update is good.